4-css-grid/styles.css

```diff
@@ -0,0 +1,18 @@
+* section {
```

Comment:
```suggestion
.section {
```
Prefer classnames over tagnames so your markup doesn't have to match your styles.